docs/KarmaCommands.md

```diff
@@ -0,0 +1,60 @@
+# Karma Command Reference
+
+Command to giveth or taketh away a user's karma
+
+## Options
+
+### For normal users
+
+#### Add 1 karma to user
+
+```sh
+/rocket @user ++
+```
+
+#### Remove 1 karma from user
+
+```sh
+/rocket @user --
```

Comment:
Pinging @bobheadxi - he mentioned something at some point about only having "add karma," not "remove karma," to encourage people to be nice. If we only have added karma, then we'd be able to simplify this to, say, `/rocket karma @user`, which I think would be a bit more consistent.

Reply:
yeah i dont think we should be allowed to remove karma - it seems like it'll only cause problems, if people take karma seriously (hopefully not overly so). Like @rwblickhan says, it'll simplify the command too

I noticed there's a `set`, I guess for situations where things are bugged and we really need to reduce someone's karma for some reason

Reply:
Ok, I agree with removing the ability to remove karma, though changing the command to add karma might not be as intuitive.

Reply:
I would really like to avoid `/rocket @user ++` if possible though - I don't think the gains from intuitiveness would be worth the loss of consistency. Is there any other way we could handle this command?

Reply:
I really like `/rocket @user ++` since the simplicity might encourage its use more, and perhaps we can add more user-user interactions via `/rocket @user` commands. I'm thinking:

```
/rocket @user are-you-coming-to-todays-meeting
/rocket @user schedule
/rocket @user ping
```

sorta reminds me of [`inertia ${remote_name} ...`](https://inertia.ubclaunchpad.com/cli/inertia_$%7Bremote_name%7D.html) 😉 

(un-resolving this conversation because i think it warrants more discussion)

Reply:
Okay, I agree with this - `/rocket @user ++` will work. I think in terms of architecture we'll want to create an `InteractionCommand` (or similar) and when we parse `@user` we can pass the target user to the `InteractionCommand`. This would enable us to extend it with more "direct interaction" features in the future (as @bobheadxi mentioned above).